src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -301,19 +312,15 @@ export const WorkspaceDashboard = _.flow(
         div({ style: { fontSize: '1rem', fontWeight: 500, marginBottom: '0.5rem' } }, [
           'Google Bucket'
         ]),
-        div({ style: { display: 'flex' } }, [
-          div({ style: Style.noWrapEllipsis }, [bucketName]),
-          h(Link, {
-            style: { margin: '0 0.5rem', flexShrink: 0 },
-            tooltip: 'Copy bucket name',
-            'aria-label': 'Copy bucket name',
-            onClick: withErrorReporting('Error copying to clipboard', async () => {
-              await clipboard.writeText(bucketName)
-              this.setState({ bucketCopied: true }, () => {
-                setTimeout(() => this.setState({ bucketCopied: undefined }), 1500)
-              })
-            })
-          }, [icon(bucketCopied ? 'check' : 'copy-to-clipboard')])
+        div({ style: { marginBottom: '0.5rem', display: 'flex' } }, [
+          div({ style: { marginRight: '0.5rem', fontWeight: 500 } }, ['Name:']),
+          h(TooltipCell, { tooltip: bucketName, style: { marginRight: '0.5rem', ...Style.noWrapEllipsis } }, [bucketName]),
+          h(ClipboardButton, { text: bucketName, style: { marginLeft: '0.25rem' } })
+        ]),
+        div({ style: { marginBottom: '0.5rem', display: 'flex' } }, [
+          div({ style: { marginRight: '0.5rem', fontWeight: 500 } }, ['Location:']),
+          div({ style: { marginRight: '0.5rem' } }, [flag || unknownRegionFlag]),
+          regionDescription || 'loading...'
```

Comment:
To me, the 'unknown' icon during load feels a bit distracting (and the text fallback isn't working as written). What about something like:
```suggestion
          bucketLocation ? h(Fragment, [
            div({ style: { marginRight: '0.5rem' } }, [flag]),
            regionDescription
          ]) : 'Loading...'
```

Reply:
Sounds good to me